nzedb/Movie.php

```diff
@@ -666,11 +668,16 @@ protected function setTmdbImdbTraktVar(&$variable1, &$variable2, &$variable3)
 	{
 		if ($this->checkVariable($variable1)) {
 			return $variable1;
-		} elseif ($this->checkVariable($variable2)) {
+		}
+
+		if ($this->checkVariable($variable2)) {
 			return $variable2;
-		} elseif ($this->checkVariable($variable3)) {
+		}
+
+		if ($this->checkVariable($variable3)) {
 			return $variable3;
 		}
+
 		return '';
```

Comment:
Please revert these changes to the IF. It needs to refactored as a switch assigning the $variableX to a dummy variable which is then returned at the end. This reduces the comlexity factor by removing the additional return points